cmd/posttriage/main.go

```diff
@@ -55,38 +56,66 @@ func main() {
 			jiraBaseURL,
 		)
 		if err != nil {
-			log.Fatalf("error building a Jira client: %v", err)
+			log.Fatalf("FATAL: error building a Jira client: %v", err)
```

Comment:
nit: Is the `FATAL:` prefix not unnecessary? I'd have expected the `logging` framework to support configuration of prefixes like this?

Reply:
yeah probably but I was lazy

Reply:
Fair :joy: